manifest/dog.glacier/ContextMenuBackOption/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "ContextMenuBackOption",
+	"id": "dog.glacier.ContextMenuBackOption",
+	"description": "Adds a Back option to every Context Menu submenu.",
+	"category": "Context Menu",
+	"sourceLocation": "https://github.com/noblereign/ResoniteContextMenuBackOption",
+	"versions": {
+		"2.3.0": {
+			"releaseUrl": "https://github.com/noblereign/ResoniteContextMenuBackOption/releases/tag/v2.3.0",
+			"artifacts": [{
+				"url": "https://github.com/noblereign/ResoniteContextMenuBackOption/releases/download/v2.3.0/ContextMenuBackOption.dll",
```

Comment:
Non-blocking note: `ResoniteMod.Debug` already won't be logged if the modloader has debug disabled. This can also be checked manually if you want other code behind the modloader debug mode with `ResoniteMod.IsDebugEnabled()`.